gatsby-node.js

```diff
@@ -31,6 +31,12 @@ exports.onCreatePage = ({ page, actions }) => {
             redirectInBrowser: true,
             isPermanent: true,
         })
+        createRedirect({
+            fromPath: `/careers/besquare/`,
+            toPath: `/careers/`,
+            redirectInBrowser: true,
+            isPermanent: true,
+        })
```

Comment:
It would be better if we have redirect for `/careers/besquare` as well.
```suggestion
        createRedirect({
            fromPath: `/careers/besquare`,
            toPath: `/careers/`,
            redirectInBrowser: true,
            isPermanent: true,
        })
        createRedirect({
            fromPath: `/careers/besquare/`,
            toPath: `/careers/`,
            redirectInBrowser: true,
            isPermanent: true,
        })
```